### PR TITLE
Refactor the local deployment documentation

### PR DESCRIPTION
**Pull request by AlphaHinex**
## Change Summary

Refactor the local deployment documentation into two sections:

1. Steps to be taken on the Internet-facing server.
2. Steps to be taken on the Intranet server.

In addition to the instructions for deploying LLM and AgentFabric, include information about deploying the agent built by AgentFabric and how to customize the configuration file paths.

## Checklist

* [x] The pull request title is a good summary of the changes - it will be used in the changelog
* [x] Unit tests for the changes exist
* [x] Run `pre-commit install` and `pre-commit run --all-files` before git commit, and passed lint check.
* [x] Some cases need DASHSCOPE_TOKEN_API to pass the Unit Tests, I have at least **pass the Unit tests on local**
* [x] Documentation reflects the changes where applicable
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

### CODE REVIEW

**Comment by AlphaHinex** [comment]
please review

File: docs/local_deploy.md [hunk 1]
```diff
@@ -1,36 +1,111 @@
...
-sudo docker pull registry.cn-hangzhou.aliyuncs.com/modelscope-repo/modelscope-agent:v0.3.0
+docker pull registry.cn-hangzhou.aliyuncs.com/modelscope-repo/modelscope-agent:v0.3.0
 ```
-#### 拉起容器
+
+导出镜像文件：
+
 ```shell
-sudo nvidia-docker run -ti --net host -v /data/work:/data/work registry.cn-hangzhou.aliyuncs.com/modelscope/modelscope-agent:v0.3.0 bash
+docker save > modelscope-agent.tar registry.cn-hangzhou.aliyuncs.com/modelscope-repo/modelscope-agent:v0.3.0
```
Comment (zzhangpurdue) [inline docs/local_deploy.md]: 这里 save > modelscope-agent.tar  是咋做什么操作？

File: docs/local_deploy.md [hunk 2]
```diff
@@ -1,36 +1,111 @@
...
+
+二、内网环境部署 LLM
+------------------
+
+### 导入镜像
+
+进入工作路径 `/data/work`，导入镜像文件：
+
+```shell
+docker load < modelscope-agent.tar
```
Comment (zzhangpurdue) [inline docs/local_deploy.md:81]: modelscope-agent.tar 这个文件是他在save的时候自动存的？

File: docs/local_deploy.md (hunk 1 above)
Comment (AlphaHinex) [reply docs/local_deploy.md]: 在外网环境把下载的镜像导出成本地文件 `/data/work/modelscope-agent.tar`，然后传输到离线环境

**Comment by zzhangpurdue** [comment]
lint check not pass

**Review by zzhangpurdue — APPROVED** [review]

**Pull request merged**